src/constants/translations/ua/accordion.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I don't think we need to use empty translations. The backup language should be EN, and at the moment we don't have the ability to switch languages.

Reply:
++